lib/chat/api/session.ts

```diff
@@ -150,6 +171,19 @@ export class SessionApi extends Construct {
                 environment: {
                     SESSIONS_TABLE_NAME: sessionTable.tableName,
                     SESSIONS_BY_USER_ID_INDEX_NAME: byUserIdIndex,
+                    GENERATED_IMAGES_S3_BUCKET_NAME: imagesBucket.bucketName
+                },
+            },
+            {
+                name: 'attach_image_to_session',
+                resource: 'session',
+                description: 'Attaches image to session',
+                path: 'session/{sessionId}/attachImage',
+                method: 'PUT',
+                environment: {
+                    SESSIONS_TABLE_NAME: sessionTable.tableName,
+                    SESSIONS_BY_USER_ID_INDEX_NAME: byUserIdIndex,
+                    GENERATED_IMAGES_S3_BUCKET_NAME: imagesBucket.bucketName
                 },
```

Comment:
Are these same 3 env variables are repeated for every lambda definition here? Should _somebody_ turn this into a variable that can be reused? 😇